src/components/AutoScroller/__tests__/util-test.ts

```diff
@@ -0,0 +1,49 @@
+import * as util from '../util';
+
+describe('AutoScroller/util', () => {
+    describe('isScrollable()', () => {
+        test.each`
+            overflow    | overflowX   | overflowY   | result
+            ${''}       | ${''}       | ${''}       | ${false}
+            ${'auto'}   | ${''}       | ${''}       | ${true}
+            ${'hidden'} | ${''}       | ${''}       | ${false}
+            ${'scroll'} | ${''}       | ${''}       | ${true}
+            ${''}       | ${'auto'}   | ${''}       | ${true}
+            ${''}       | ${'auto'}   | ${''}       | ${true}
+            ${''}       | ${'scroll'} | ${''}       | ${true}
+            ${''}       | ${'hidden'} | ${''}       | ${false}
+            ${''}       | ${'hidden'} | ${'auto'}   | ${true}
+            ${''}       | ${''}       | ${'auto'}   | ${true}
+            ${''}       | ${''}       | ${'scroll'} | ${true}
+            ${''}       | ${''}       | ${'hidden'} | ${false}
+            ${''}       | ${'auto'}   | ${'hidden'} | ${true}
+        `('should return %s based on the provided element', ({ overflow, overflowX, overflowY, result }) => {
```

Comment:
What does %s use here? Should it be `$result`?

Reply:
Good catch.